test/threadpool/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
-VIGRA_ADD_TEST(test_threadpool test.cxx)
-
+if (WITH_BOOST_THREAD)
```

Comment:
@ukoethe are there other places where additional linking to ``boost_thread`` and ``boost_system`` is needed, or is this handled automatically?